examples/vpc_service_control/vpc_sc_project/terraform.tfvars

```diff
@@ -0,0 +1,8 @@
+project_id = ""
+service_account_file = ".json"
+organization_id = ""
+billing_account_id = ""
+cloud_router_region = "us-west1"
+shared_secret_string_for_vpn_connection = ""
+ip_addr_of_onprem_vpn_router = ""
```

Comment:
```suggestion
```

Reply:
Will delete the tfvars file.

Reply:
No need to delete the tfvars file. You can simply create a terraform.tfvars.example file as mentioned [here](https://github.com/terraform-google-modules/terraform-google-vpc-service-controls/pull/4#discussion_r295596474)